wallet.md

```diff
@@ -79,7 +79,7 @@ Follow the steps in [this guide](backup.md) to backup your wallet and to restore
 
 By default your wallet is pre-configured with one main account. This is sufficient for performing basic transactions and for receiving coin awards for running a full node.
 
-You can add up to 256 additional accounts to your wallet.
+You can add up to 256 additional accounts to your wallet. 
```

Comment:
Is the total 256 or 257?

Reply:
I need to double-check with @IlyaVi

Reply:
Ilya says that currently there is no limit, so we should remove this sentence.

Reply:
yep, doing this now.